web/templates/page/rules.html.eex

```diff
@@ -0,0 +1,24 @@
+<h2>CMM Ping Pong Rules</h2>
+
+<div id="rules">
+  <h3>Volley</h3>
+  <p>
+    Players must volley before the match to decide who serves first. The ball must be hit a minimum of 4 times. The player who gets the point starts the match serving.
+  </p>
+  <h3>Legal Serve</h3>
+  <p>
+    The ball must rest on an open hand palm. Then it must be tossed up at least 6 inches and struck so the ball first bounces on the server's side and then the opponent's side.
+  </p>
+  <p>
+    If the serve is legal except that it touches the net, it is called a let serve. Let serves are not scored and are reserved.
+  </p>
+  <h3>Alternate Serving</h3>
+  <p>
+    After each 5 points have been scored the receiving player or pair shall become the serving player or pair and so on until the end of the game, unless both players or pairs score 20 points or the expedite system is in operation, when the sequences of serving and receiving shall be the same but each player shall serve for only 1 point in turn.
```

Comment:
These sound super legal/formal. Should we make the text of the rules more user-friendly?